addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1549,6 +1525,17 @@ class UnitUserValue extends UserValue {
             || !!this.value.match(/\d+\s+\d+/)
         );
     }
+    /**
+     * @override
+     */
+    loadMethodsData() {
+        super.loadMethodsData(...arguments);
+        const params = this._methodsParams;
+        const unit = params.unit || '';
+        if (params.saveUnit === undefined) {
+            params.saveUnit = unit;
+        }
```

Comment:
Was this related to the props thing that led to a bug? Should I squash that fix into this one?

Reply:
Yes, ultimately they should all be squashed into the WeInput commit.